modules/social_features/social_tagging/social_tagging.module

```diff
@@ -105,10 +106,32 @@ function social_tagging_form_taxonomy_overview_terms_alter(&$form, FormStateInte
  * Implements hook_form_FORM_ID_alter().
  */
 function social_tagging_form_node_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+
+  /** @var \Drupal\node\Entity\NodeType $node_type */
+  $node_type = \Drupal::routeMatch()->getParameter('node_type');
+  /** @var \Drupal\node\Entity\Node $node */
+  $node = \Drupal::routeMatch()->getParameter('node');
+
+  // On node edit we need to do this.
+  if (!$node_type) {
+    $node_type = NodeType::load($node->bundle());
+  }
+
+  $node_type_tagging = TRUE;
+  if ($node_type instanceof NodeType) {
+
+    $config = \Drupal::getContainer()->get('config.factory')->getEditable('social_tagging.settings');
+    $config_value = $config->get('tag_node_type_' . $node_type->id());
+
+    if ($config_value === FALSE || $config_value === 0) {
+      $node_type_tagging = FALSE;
+    }
+  }
+
   // Get the taggingservice.
   $tag_service = Drupal::getContainer()->get('social_tagging.tag_service');
   // Check if tagging is turned on/off.
-  if (!$tag_service->active()) {
+  if (!$tag_service->active() || !$tag_service->hasContent() || $node_type_tagging === FALSE) {
```

Comment:
Why did you add `!$tag_service->hasContent()` that makes no sense to me? That means that the field will never be shown when there is no tagged content? So how do you create your first content item with tags?